drivers/bluetooth/hci/hci_ifx_cyw208xx.c

```diff
@@ -303,12 +328,10 @@ static int cyw208xx_send(const struct device *dev, struct net_buf *buf)
 	return ret ? -EIO : 0;
 }
 
-static DEVICE_API(bt_hci, drv) = {
-	.open = cyw208xx_open,
-	.close = cyw208xx_close,
-	.send = cyw208xx_send,
-	.setup = cyw208xx_setup
-};
+static DEVICE_API(bt_hci, drv) = {.open = cyw208xx_open,
+				  .close = cyw208xx_close,
+				  .send = cyw208xx_send,
+				  .setup = cyw208xx_setup};
```

Comment:
This doesn't look like a relevant change, or even a good one. IMO the original style is much clearer.